actions/build-ml-whl/action.yaml

```diff
@@ -43,28 +44,32 @@ runs:
         else
             export BUILD_TYPE="nightly"
         fi
+        echo "BUILD_TYPE=${BUILD_TYPE}"
+
         makefile_path=$(find . -type f -name "Makefile")
         cd $(dirname ${makefile_path})
+
         status=0
-        BUILD_TYPE=$BUILD_TYPE python -m build . || status=$?
+        make build || status=$?
```

Comment:
Actually based on the Makefile, the command should be:

`make build BUILD_TYPE=$BUILD_TYPE`

Reply:
The output from this run indicates that the make command correctly inherits the `BUILD_TYPE` env var: https://github.com/neuralmagic/compressed-tensors/actions/runs/14601377079/job/40960082433

Specifically:

```
Building the wheel for the repository
BUILD_TYPE=nightly python3 setup.py sdist bdist_wheel;
```

However, it's still producing a dev version.

Reply:
@dhuangnm I’ve update the description – the link at the bottom of it is a new run from today with the current version of the branch using the `ubuntu-22.04` runner (the same as we currently have in, e.g., `compressed-tensors`). That run passes.

It’s worth noting that running the same everything with the newer GitHub-hosted Ubuntu runner (either `ubuntu-24.04` or `ubuntu-latest`, both pointing to the same image right now), it fails. Our `beaker` machine is also running `22.04`, so it seems that whatever changes are in `setuptools-scm` 8.3.1 compared to 8.3.0 "fix" something in 22.04 but not 24.04 environments – perhaps a puzzle for another day.

Reply:
In the last two nightly runs, the day that installed 8.3.0 worked as expected and the next day that installed 8.3.1 generated the dev build. As such, for the time being at least, I update this PR to pin explicitly to `==8.3.0`.